Zend/tests/object_types/return_type_in_interface.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+Adding a interface method object return type
+--FILE--
+<?php
+
+interface One {
+    public function a() : object;
+}
+
+$two = new class implements One {
+    public function a() : object {
+        return 12345;
+    }
+};
+$two->a();
+--EXPECTF--
+
+Fatal error: Uncaught TypeError: Return value of class@anonymous::a() must be an object, integer returned in %s:9
+Stack trace:
+#0 %s(12): class@anonymous->a()
+#1 {main}
+  thrown in %s on line 9
```

Comment:
This test looks redundant with Zend/tests/object_types/return_type_in_class.phpt, which also has an object type on the interface.